src/main/java/com/softserveinc/ita/utils/runners/TestRunner.java

```diff
@@ -6,7 +6,6 @@
 import static com.codeborne.selenide.Selenide.open;
 
 public abstract class TestRunner {
-
     protected HomePage homePage = new HomePage();
```

Comment:
let's stop initializing pages in test runner

Reply:
done